.github/workflows/validate_packs.yml

```diff
@@ -306,3 +306,51 @@ jobs:
               "target_url": "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}",
               "context": "Validation / Install plan ids exist"
             }'
+ensure-quickstart-dashboard-names-are-unique:
```

Comment:
The formatting for this needs to be updated! this should be indented inwards and there should be a line after 308